.github/workflows/v2.yaml

```diff
@@ -26,8 +27,6 @@ jobs:
         docker build \
         --pull \
         --no-cache \
-        --tag composer/composer:latest \
```

Comment:
I think it doesn't make sense to push those tags for non-latest versions

Reply:
You're right, it does not.